src/test/java/seedu/club/commons/util/XmlUtilTest.java

```diff
@@ -84,6 +84,8 @@ public void xmlAdaptedMemberFromFile_fileWithMissingMemberField_validResult() th
                 MISSING_MEMBER_FIELD_FILE, XmlAdaptedMemberWithRootElement.class);
         XmlAdaptedMember expectedMember = new XmlAdaptedMember(
                 null, VALID_PHONE, VALID_EMAIL, VALID_MATRIC_NUMBER, VALID_GROUP, VALID_TAGS);
+        System.out.println("expectedMember.getPassword() = " + expectedMember.getPassword());
+        System.out.println("actualMember = " + actualMember.getPassword());
```

Comment:
Remember to remove them before submission on Sunday.. i'm not sure if we should be logging passwords 😅